src/QMCDrivers/VMC/VMCDriverInput.cpp

```diff
@@ -26,6 +26,8 @@ void VMCDriverInput::readXML(xmlNodePtr node)
   parameter_set_.add(samples_, "samples", "int");
   parameter_set_.add(samples_per_thread_, "samplesperthread", "int");
   parameter_set_.add(steps_between_samples_, "stepsbetweensamples", "int");
+  parameter_set_.add(total_walkers_, "walkers", "int");  
+  parameter_set_.add(total_walkers_, "total_walkers", "int");  
```

Comment:
It is better not to change the definition of existing input tags. Keep `walkers` as the number of walkers per rank.
Move "total_walkers" to base input class and use it for both VMC and DMC.